runtime/doc/builtin.txt

```diff
@@ -1847,6 +1847,7 @@ execute({command} [, {silent}])				*execute()*
 <			foo >
 			echo execute(['echon "foo"', 'echon "bar"'])
 <			foobar
+<		Except that line continuation in {command} is not recognized.
```

Comment:
```suggestion
		Line continuations in {command} are not recognized.
```

Reply:
Maybe also move this before the examples.